### ISSUE TITLE
YAML Support

### ISSUE THREAD

**Issue by helloanoop** [usebruno/bruno#3577]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Overview
This effort is to track `.yml` as the preferred format for saving API request data instead of `.bru`. For the progression of the Domain-Specific Language (DSL), see https://github.com/usebruno/bruno/discussions/360

### Why YAML
- Well established format
- Utilize syntax highlighting in IDEs
- Leverage existing tooling for toml file manipulation
- Git providers (like GitHub, Bitbucket) offer syntax highlighting while reviewing pull requests on the web

### Mockups or Images of the feature

None

**Comment by fantpmas** [comment]
Any view on the migration path for `.bru` files? Thanks!

**Comment by helloanoop** [comment]
@fantpmas Yes, we will build a migration path as well as support both formats for a long time.

**Comment by anusree-bruno** [comment]
Update on this:
We’re working on a separate file storage package [PR #5130](https://github.com/usebruno/bruno/pull/5130) for setting things up for YAML support. It adds an abstraction layer so we can support formats beyond .bru (like YAML) without needing a bunch of changes.

**Comment by sreelakshmi-bruno** [comment]
Hi @fantpmas , we would like to talk to you and get some feedback about YAML support . If you're interested, please select your [preferred time here](https://cal.com/sreelakshmi-jayarajan/product-discovery). Thanks!

**Comment by lucksidedown** [comment]
Can I sign-up to beta test this when it's available to beta test?

**Comment by Its-treason** [comment]
> Can I sign-up to beta test this when it's available to beta test?

@lucksidedown You can already download the preview builds here: https://github.com/usebruno/bruno-nightly-builds

**Comment by trymeouteh** [comment]
Please consider adding TOML format option if possible

**Comment by helloanoop** [comment]
Bruno v3 now supports YAML 🎉 🎉 
Download the latest at https://www.usebruno.com/downloads
Join the discussion at https://github.com/usebruno/bruno/discussions/6634

**Comment by shfx** [comment]
We don't have support for open collection yaml files in `bru run`, do I understand that correctly?

**Comment by benc-uk** [comment]
I just tried `bru run` with a OpenCollection yaml and it worked fine. To my surprise I might add!

**Comment by shfx** [comment]
Interesting. Tried it once and it didn't recognise yml files. Will check again soon.
